source/developer/personal-access-tokens.rst

```diff
@@ -0,0 +1,92 @@
+
+3 - To create an access token with a non-admin account, you must first give it the appropriate permissions. Go to **System Console > Users**, search for the user account, and select **Manage Roles** from the dropdown.
+
+	.. image:: ../../source/images/access_token_manage_roles.png
+
+4 - Select **Allow this account to generate user access tokens.**
+
+	.. image:: ../../source/images/access_tokens_additional_roles.png
+  
+  // XXX This image needs UI tweaks from Asaad
+
+You may optionally allow the account to post to any channel in your Mattermost server, including direct messages by choosing the **post:all** role. **post:channels** role allows the account to post to any public channel in the Mattermost server.
+
+Then hit **Save**.
+
+5 - Sign in to the user account to create a personal access token.
+
+6 - Go to **Account Settings > Security > Personal Access Tokens** and click **Create New Token**.
+
+  // XXX Take a screenshot
+
+7 - Enter a description for the token, so you remember what it's used for. Then hit **Save**.
+
+.. note::
+  If you create a personal access token for a System Admin account, be careful who you share it with. The token enables a user to have full access to the account, including System Admin privileges.
+  
+  It is recommended to create a personal access token for non-admin accounts.
+  
+8 - Copy the access token now for your integration and store it in a secure location. You won't be able to see it again!
+
+9 - You're all set! You can now use the personal access token for integrations to interact with your Mattermost server and `authenticate against the Mattermost REST API <https://about.mattermost.com/default-api-authentication>`_.
+
+Revoking a Personal Access Token
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A personal access token can be revoked by deleting the token from either user's account settings or from the System Console.
+
+Once deleted, all sessions using the token are deleted and any attempts to interact with the Mattermost server are blocked.
+
+Account Settings
+.................
+
+1 - Sign in to the user account and go to **Account Settings > Security > Personal Access Tokens**. 
+
+2 - Identify the access token you want to revoke and hit **Delete**. Confirm the deletion.
+
+System Console
+.................
+
+1 - Go to **System Console > Users**, search for the user account which the token belongs to, and select **Manage Tokens** from the dropdown.
+
+2 - Identify the access token you want to revoke and hit **Delete**. Confirm the deletion.
+
+Frequently Asked Questions (FAQ)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+How do personal access tokens differ from regular session tokens?
+..................................................................
+
+Personal access tokens last forever. As a result, you can more easily integrate with Mattermost, bypassing the `session length limits set in the System Console <https://docs.mattermost.com/administration/config-settings.html#sessions>`_.
```

Comment:
Per my above comment, probably better to say they don't expire rather than last forever